playwright/tests/organize/journeys/journey-instance/sidebar.spec.ts

```diff
@@ -293,7 +299,12 @@ test.describe('Journey instance sidebar', () => {
       //find x-icon and click it, then wait for re-fetch of invalidated instance data
       await Promise.all([
         page.waitForResponse(
-          `**/orgs/${KPD.id}/journey_instances/${ClarasOnboarding.id}`
+          (res) =>
+            res.request().method() === 'GET' &&
+            res
+              .request()
+              .url()
+              .endsWith(`/journey_instances/${ClarasOnboarding.id}`)
```

Comment:
This is necessary, and better IMO than what's on line 148. So I think both of those cases should use this pattern, where we're waiting for the `GET` initiated by react-query cache invalidation after the `DELETE` has already completed.